src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/health/HealthCheck.java

```diff
@@ -29,10 +29,10 @@ public Health health() {
                             .block(timeout);
             return Health.up().withDetail("HttpStatus", responseEntity.getStatusCode()).build();
         } catch (final WebClientResponseException e) {
-            log.error("Dependant health check failed", e);
+            log.error("Dependant health check failed in class: " + this.getClass(), e);
             return Health.down(e).withDetail("body", e.getResponseBodyAsString()).build();
         } catch (final Exception e) {
-            log.error("Dependant health check failed", e);
+            log.error("Dependant health check failed in class: " + this.getClass(), e);
```

Comment:
Also, don't you want to distinguish between WebClient and other exceptions... just a slightly different log string?